project/frontend_views_and_designs/yarn.lock

```diff
@@ -0,0 +1,3818 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.

```

Comment:
What this file is for?

Reply:
It is a part of node config like package-lock
I use Yarn instead of npm(no big differences) 
It contain dependecy and node module package versionsm